### ISSUE TITLE
[document]What do `GKs` and `gkx` mean?

### ISSUE THREAD

**Issue by AkifumiSato** [facebookexperimental/Recoil#2141]
- Does `GK` stand for `GateKeeper`?
- Are these feature flags?
- Will the `RecoilEnv` documentation be updated in the next release to include a note about `GKs`?

**Comment by mondaychen** [comment]
Yes and yes. We use gate keeper instead of feature flag since some of them are internal implementation details, not features.
Thanks for raising this issue. We'll update the doc to include more information next time we release.
